README.md

```diff
@@ -70,6 +70,29 @@ animation.apply_to_tween(tween, self)
 ```
 The method takes a Tween to which you want to apply the animation and a Node that will be root of the animation.
 
+#### Parameters
+You can also add/remove parameters to be able to target them in the editor with a specific syntax described in the "Value fields" section below.
+Useful to be able to assign custom values per animation instead of having to create properties or meta data

```

Comment:
`set_parameter()` will apply parameter to all instances of an animation (because Resources are shared by default). Is this expected? Maybe it should be noted/clarified.

Reply:
I will clarify it somehow. For me it's a given as resources work that way, and users can make duplicates in code if they wish so.